cfme/tests/cloud_infra_common/test_rest_providers.py

```diff
@@ -116,6 +120,7 @@ def test_provider_delete_from_detail(provider_rest, appliance, method):
     assert_response(appliance, http_status=404)
 
 
+@pytest.mark.rhv1
```

Comment:
Delete provider is a very basic operation that needs to be include in tier 1.
You are right that those tests have the same functionality:
I picked one to be tier 1 (this will verify the functionality of provider removal work), and put the other in tier 3.

Reply:
Delete provider is important & basic.
est_provider_delete_from_detail  & test_provider_delete_from_collection have same functionality (they differ in the UI part). This is why I picked one of them (test_provider_delete_from_collection) to be a representative test, and marked the second (test_provider_delete_from_detail) as T3.

Reply:
Okay, I understand the reasoning. If that is the case however, I suggest switching the tests. Let's make delete provider from UI part of T1, since CFME is primarily focused on UI functionality. Everything should be possible to achieve from UI - REST is only "bonus" let's say. I know REST test is probably quicker. But if the deletion from UI is broken, is much bigger problem than if the deletion from REST is broken.

Reply:
ok. will switch the ties between those 2 tests.